hijack/views.py

```diff
@@ -93,32 +92,34 @@ def dispatch(self, request, *args, **kwargs):
 
     @method_decorator(csrf_protect)
     def post(self, request, *args, **kwargs):
-        hijacker = request.user
-        hijacked = self.get_object()
+        with transaction.atomic():
```

Comment:
Would you like to use it as a decorator?

Reply:
Hi @Mogost, thanks for your feedback. I have added trasaction.atomic as a decorator. Please review. Thanks